2996_reflection/reflection.md

```diff
@@ -3142,12 +3230,24 @@ Add a bullet to paragraph 13:
 [13]{.pnum} A declaration `$D$` _names_ an entity `$E$` if
 
 * [13.1]{.pnum} `$D$` contains a `$lambda-expression$` whose closure type is `$E$`,
-* [13.1+]{.pnum} [`$D$` contains an expression of type `std::meta::info` that represents either `$E$` or a type alias or namespace alias that denotes `$E$`,]{.addu}
+* [13.1+]{.pnum} [`$D$` contains an expression of type `std::meta::info` that represents an entity that is either `$E$` or whose underlying entity is `$E$`,]{.addu}
```

Comment:
So it's intentional that `using T = C;` names both `T` and `C`? Just want to check.

Reply:
On second thought - No, it only needs to name `T` (thanks for the sanity check).

The notion of when a declaration _names_ an entity is mostly used to define when a declaration is an exposure (i.e., when it names a TU-local entity). But since any type alias or namespace alias is already TU-local, any declaration that names the alias is already an exposure - so no need to really pay any attention to the underlying entity.

Note - This section needs some more attention and scrutiny. CWG asked for examples of what I'm trying to achieve here in Poland; I still need to get around to that. I'll probably turn to that after this PR is merged.